rules/DowngradePhp73/Rector/FuncCall/SetCookieOptionsArrayToArgumentsRector.php

```diff
@@ -118,24 +108,20 @@ private function composeNewArgs(FuncCall $funcCall): array
     {
         $this->highestIndex = 1;
 
-        if (! $this->argsAnalyzer->isArgsInstanceInArgsPositions($funcCall->args, [0, 1, 2])) {
+        if (count($funcCall->getArgs()) < 3) {
```

Comment:
No, `getArgs()` throw warning too early, see 

https://github.com/nikic/PHP-Parser/blob/0ffddce52d816f72d0efc4d9b02e276d3309ef01/lib/PhpParser/Node/Expr/CallLike.php#L36

Reply:
I see. I think these asserts are never executed, never came across it.

Reply:
I've added a fixture to covert it with (...) and is passed. Could you share a PR with failing test fixture? 

I'll merge this, so we can iterate to improve.

Reply:
while downgrade rule always downgrade `DowngradeFirstClassCallableSyntaxRector` early, this may cause jump rule which the args still a first class callable, that's wil cause error:

```bash
PHP Warning:  Uncaught AssertionError: assert(!$this->isFirstClassCallable()) in php shell code:1
```

Reply:
Assert is show or not depends on [zend.assertions](https://www.php.net/manual/en/ini.core.php#ini.zend.assertions). setting, which default OFF on our CI.